src/conditional-flow/multiple-conditions.js

```diff
@@ -31,8 +41,42 @@ function isHelloOrGoodbye (val1) {
 function getAgeDescription (age) {
 
   // TODO: write code in this function body to pass the tests
+  switch(age) {
+    case 0:
+      Text = "Baby";
+      break;
+    case 1: 
+    case 2:
+    case 3:
+    case 4:
```

Comment:
switch is possibly not best option here.
Could use if ( isInRange( age, 1, 4 ) ) Text = "Baby";
with else ifs etc